runtime/doc/index.txt

```diff
@@ -339,7 +339,7 @@ tag		char	      note action in Normal mode	~
 				   insert text, repeat N times
 |P|		["x]P		2  put the text [from register x] before the
 				   cursor N times
-|Q|		Q		   switch to "Ex" mode
+|gQ|		gQ		   switch to "Ex" mode
```

Comment:
Remove this line. This is already present. Just search for `gQ` in `index.txt`.

Reply:
Woops, by bad, removed in adefbcb8e724a4a917a8505e598ac47164be0858